ds/beis_indicators/utils/dir_file_management.py

```diff
@@ -50,13 +47,12 @@ def parse_academic_year(year):
     '''
     return(int(year.split('/')[0]))
 
-def make_indicator(table,target_path,var_lookup,year_var,nuts_var='nuts_code',nuts_spec='flex',decimals=0):
+def make_indicator(table,var_lookup,year_var,nuts_var='nuts_code',nuts_spec='flex',decimals=0):
```

Comment:
Nice, given that this is closing #185 too, I'd mentioning it in the commit message.

I also guess we should: 
- update all the files using it in `ds/beis_indicators`
- in all notebooks in `ds/notebooks`: import it and remove the cells with its old definition.

Correct?